doc/radosgw/dynamicresharding.rst

```diff
@@ -185,27 +190,28 @@ Troubleshooting
 
 Clusters prior to Luminous 12.2.11 and Mimic 13.2.5 left behind stale bucket
 instance entries, which were not automatically cleaned up. This issue also affected
-LifeCycle policies, which were no longer applied to resharded buckets. Both of
+lifecycle policies, which were no longer applied to resharded buckets. Both of
 these issues could be worked around by running ``radosgw-admin`` commands.
 
-Stale instance management
+Stale Instance Management
 -------------------------
 
-List the stale instances in a cluster that are ready to be cleaned up.
+List the stale instances in a cluster that are ready to be cleaned up:
+
+.. prompt:: bash #
 
-::
+   radosgw-admin reshard stale-instances list
 
-   # radosgw-admin reshard stale-instances list
+Clean up the stale instances in a cluster:
 
-Clean up the stale instances in a cluster. Note: cleanup of these
-instances should only be done on a single-site cluster.
+.. prompt:: bash #
 
-::
+   radosgw-admin reshard stale-instances delete
 
-   # radosgw-admin reshard stale-instances delete
+.. note:: Cleanup of stale instances should only be done on a single-site cluster.
```

Comment:
This should be clarified, single-site kinda implies "not stretch" to me, but I suspect you mean a cluster that is not using RGW multisite (async) replication? It's easy for folks to confuse the two so I like to be explicit.

Reply:
Will be happy to improve this, thank you.

I _suspect_ it indeed may be related to multisite. https://ceph--63163.org.readthedocs.build/en/63163/radosgw/dynamicresharding/#multisite talks about dynamic resharding not being supported on multisite; https://ceph--63163.org.readthedocs.build/en/63163/radosgw/zone-features/#feature-resharding seems to clarify that prior to Reef replication would be "interrupted".  
Of course cleanup might have some other, possibly worse, effects and I could not say what exactly or test it.

I can change the text for example to the opposite wording i.e. "Cleanup of stale instances should not be done in a multisite environment."

Reply:
Changed the language.